cloudformation/scylla.yaml

```diff
@@ -0,0 +1,591 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: >-
+  AWS CloudFormation Scylla Sample Template: This would create a new Scylla Cluster
+
+Parameters:
+  ClusterName:
+    Type: 'String'
+
+  InstanceCount:
+    Description: Number of EC2 instances (must be between 1 and 10).
+    Type: Number
+    Default: 1
+    MinValue: 1
+    MaxValue: 10
+    ConstraintDescription: Must be a number between 1 and 10.
+
+  ScyllaVersion:
+    Type: String
+    Default: ScyllaDB310rc3
+    AllowedValues:
+      - ScyllaDB310rc3
+
+  InstanceType:
+    Description: EC2 instance type
+    Type: String
+    Default: i3.large
+    AllowedValues:
+      - i3.large
+      - i3.xlarge
+      - i3.2xlarge
+      - i3.4xlarge
+      - i3.8xlarge
+      - i3.16xlarge
+      - i3en.large
+      - i3en.xlarge
+      - i3en.2xlarge
+      - i3en.3xlarge
+      - i3en.6xlarge
+      - i3en.12xlarge
+      - i3en.24xlarge
+      - i3en.metal
+    ConstraintDescription: must be a valid EC2 instance type.
+
+  AvailabilityZone:
+    Type: 'AWS::EC2::AvailabilityZone::Name'
+    ConstraintDescription: must be the name of available AvailabilityZone.
+
+  KeyName:
+    Description: Name of an existing EC2 KeyPair to enable SSH access to the instances
+    Type: 'AWS::EC2::KeyPair::KeyName'
+    ConstraintDescription: must be the name of an existing EC2 KeyPair.
+
+Mappings:
+  AWSRegionVersion2AMI:
+    us-east-1:
+      ScyllaDB310rc3: ami-046e87cc62fb6af2f
```

Comment:
same question
Why not making the AMI and region input parameters

Reply:
removed the hardcoded ami